### PR TITLE
Create ui-c8y-1020-26-4-use-new-object-mapping-component-for-opc-ua-6834-graft-release.md

### PR DESCRIPTION
**Pull request by gen-App-Enablement**
Release note created by c8y-ui-automations[bot]

Ticket: [DM-3561](https://cumulocity.atlassian.net/browse/DM-3561)
Original PR: [7049](https://github.softwareag.com/IOTA/cumulocity-ui/pull/7049)

### CODE REVIEW

File: content/change-logs/device-management/ui-c8y-1020-26-4-use-new-object-mapping-component-for-opc-ua-6834-graft-release.md [hunk 1]
```diff
@@ -0,0 +1,17 @@
+---
+date: ""
+title: use new object-mapping component for OPC-UA (#6834) [GRAFT][release/cd] (#7049)
```
Comment (gen-App-Enablement) [inline content/change-logs/device-management/ui-c8y-1020-26-4-use-new-object-mapping-component-for-opc-ua-6834-graft-release.md:3]: The Claude AI suggested to use the following title:
```suggestion
title: Object mapping for OPC-UA uses new component
```

File: content/change-logs/device-management/ui-c8y-1020-26-4-use-new-object-mapping-component-for-opc-ua-6834-graft-release.md [hunk 2]
```diff
@@ -0,0 +1,17 @@
...
+component:
+  - value: component-Tf05_KQ-B
+    label: OPC UA
+build_artifact:
+  - value: tc-pjJiURv9Y
+    label: ui-c8y
+ticket: DM-3561
+version: 1020.26.4
+---
+use new object-mapping component for OPC-UA (#6834) [GRAFT][release/cd] (#7049)
```
Comment (gen-App-Enablement) [inline content/change-logs/device-management/ui-c8y-1020-26-4-use-new-object-mapping-component-for-opc-ua-6834-graft-release.md:17]: The Claude AI suggested to use the following body:
```suggestion
As part of the ongoing improvements to the OPC-UA protocol adapter, the object mapping functionality now uses a new component. This change aligns the OPC-UA adapter with the other protocol adapters, making the functionality more consistent across the different adapters. For users of the OPC-UA protocol adapter, the object mapping configuration and behavior remains unchanged.
```

**Review by gen-App-Enablement — COMMENTED** [review]
AI improvements

File: content/change-logs/device-management/ui-c8y-1020-26-4-use-new-object-mapping-component-for-opc-ua-6834-graft-release.md (hunk 2 above)
Comment (dominikschmitz) [reply content/change-logs/device-management/ui-c8y-1020-26-4-use-new-object-mapping-component-for-opc-ua-6834-graft-release.md:17]:
```suggestion
We have removed the 100-character limit for text fields in the 'Send Measurement,' 'Create Alarm,' and 'Send Event' functionalities, allowing you to enter longer text without restrictions.
```

File: content/change-logs/device-management/ui-c8y-1020-26-4-use-new-object-mapping-component-for-opc-ua-6834-graft-release.md [hunk 3]
```diff
@@ -0,0 +1,17 @@
+---
+date: ""
+title: The character limit is removed in OPC-UA object-mapping functionality
```
Comment (BeateRixen) [inline content/change-logs/device-management/ui-c8y-1020-26-4-use-new-object-mapping-component-for-opc-ua-6834-graft-release.md:3]:
```suggestion
title: Character limit removed in OPC UA object-mapping functionalities
```

File: content/change-logs/device-management/ui-c8y-1020-26-4-use-new-object-mapping-component-for-opc-ua-6834-graft-release.md [hunk 4]
```diff
@@ -0,0 +1,17 @@
...
+component:
+  - value: component-Tf05_KQ-B
+    label: OPC UA
+build_artifact:
+  - value: tc-pjJiURv9Y
+    label: ui-c8y
+ticket: DM-3561
+version: 1020.26.4
+---
+We have removed the 100-character limit for text fields in the 'Send Measurement,' 'Create Alarm,' and 'Send Event' functionalities, allowing you to enter longer text without restrictions.
```
Comment (BeateRixen) [inline content/change-logs/device-management/ui-c8y-1020-26-4-use-new-object-mapping-component-for-opc-ua-6834-graft-release.md:17]:
```suggestion
The text fields in the "Send measurement", "Create alarm", and "Send event" functionalities used to be limited to 100 characters. This limitation has been removed, allowing you to enter longer text without restrictions.
```

**Review by dominikschmitz — APPROVED** [review]

File: content/change-logs/device-management/ui-c8y-1020-26-4-use-new-object-mapping-component-for-opc-ua-6834-graft-release.md [hunk 5]
```diff
@@ -0,0 +1,17 @@
+---
+date: ""
+title: Character limit removed in OPC UA object-mapping functionalities
+product_area: Device management & connectivity
+change_type:
+  - value: change-VSkj2iV9m
+    label: Fix
```
Comment (Oezge-Akin) [inline content/change-logs/device-management/ui-c8y-1020-26-4-use-new-object-mapping-component-for-opc-ua-6834-graft-release.md]:
```suggestion
change_type:
  - value: change-2c7RdTdXo4
    label: Improvement
```

**Review by Oezge-Akin — APPROVED** [review]

**Review by aaronraab — APPROVED** [review]

**Review by MWindel — APPROVED** [review]

**Pull request merged**